here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/DefaultStorageHandler.java

```diff
@@ -103,7 +108,7 @@ public DefaultStorageHandler(
     addStorageIdToStreamInfo(storageId, ctx);
 
     // Obtain IStorage implementation using NakshaHub
-    final IStorage storageImpl = nakshaHub().getStorageById(storageId);
+    final IStorage storageImpl = nakshaHub.getStorageById(storageId);
```

Comment:
reason why I usually prefer method call (like nakshaHub()) instead of directly accessing nakshaHub variable is, to allow us to transparently add more logic in the function in future, without breaking semantics. So, unless there was any specific reason in this PR to bypass it?, I suggest we keep it a function call, as was before

Reply:
Sure thing, will revert this